tests/nvidianetwork/deploy-nno-test.go

```diff
@@ -301,9 +286,9 @@ var _ = Describe("NNO", Ordered, Label(tsparams.LabelSuite), func() {
 
 				} else {
 					glog.V(networkparams.LogLevel).Infof("The nfd packagemanifest '%s' was found in the "+
-						"default catalog '%s'", nfdPkgManifestBuilderByCatalog.Object.Name, nfdCatalogSourceDefault)
+						"default catalog '%s'", nfdPkgManifestBuilderByCatalog.Object.Name, nfd.CatalogSourceNamespace)
```

Comment:
This should be `nfd.CatalogSourceDefault` otherwise the Subscription will fail to be created.